charts/incubator/wg-easy/values.yaml

```diff
@@ -0,0 +1,41 @@
+image:
+  repository: https://hub.docker.com/r/weejewel/wg-easy
+  pullPolicy: IfNotPresent
+  tag: sha256-3ff40254ba06cc93c2f3c348197e0eceac4bf4e5fee04e44d01c38daebc7631c
+
+securityContext:
+  readOnlyRootFilesystem: false
+  capabilities:
+    add:
+    - NET_ADMIN
+    - SYS_MODULE
+  runAsNonRoot: false
+    sysctls:
+        - name: net.ipv4.conf.all.src_valid_mark
+            value: '1'
+
+podSecurityContext:
+  runAsUser: 0
+  runAsGroup: 0
+
+service:
+  main:
+    protocol: TCP
```

Comment:
```suggestion
```
You can omit this, this is the default ;)

Reply:
This I didn't know lol